packages/core/ui-settings/core-ui-settings-server-internal/src/types.ts

```diff
@@ -11,6 +11,9 @@ import type {
   UiSettingsServiceSetup,
   UiSettingsServiceStart,
 } from '@kbn/core-ui-settings-server';
+import { SavedObjectsClientContract } from '@kbn/core-saved-objects-api-server';
+import { UiSettingsParams } from '@kbn/core-ui-settings-common';
+import type { Logger } from '@kbn/logging';
```

Comment:
NIT: the previous `NIT: import type` was about all these 3 lines 😄

Reply:
Will address in the next PR 😬